docs/user_guide/nvflare_cli/fl_simulator.rst

```diff
@@ -837,6 +837,11 @@ Specifying threads
 ==================
 The simulator ``-t`` option provides the ability to specify how many threads to run the simulator with.
 
+.. note::
+
+    We use the term threads for simplicity, however technically each client actually runs in a separate process.
```

Comment:
I agree we should, I put this note here for now since there's several other places that would need changes for consistency, eg:
`nvflare simulator -h ...  
-t THREADS, --threads THREADS
                        number of parallel running clients`
                        
Since Yuhong will be redesigning the simulator maybe we can update everything accordingly then?